scripts/localnets/cosmoshub/.cosmoshub/config/node_key.json

```diff
@@ -0,0 +1 @@
+{"priv_key":{"type":"tendermint/PrivKeyEd25519","value":"PAshqNauFTLl5nK4KLTRRT/3kDs1wJhvKDgGF6JI/zyajrTzr0gnPtPkrhgRfKziMhfmVFZ2NghkuVvvTHltCA=="}}
```

Comment:
In the builder repo we generate keys on the fly, and output them to a temporary file for tests to gain access to them. Encoding them directly creates a lot of boilerplate code/files for us to maintain, is there a better way to do it here?